app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,8 @@
     <%= Gon::Base.render_data %>
   </head>
   <body>
+    <%= content_for :render_async %>
```

Comment:
hmm, definitely not consistent, the js gets generated with a securerandom. https://github.com/kaspergrubbe/render_async/blob/0e11963c9d13df5d266b79044934cbf649dafb2a/lib/render_async/view_helper.rb#L6

Maybe nonces are the way to go here? (maybe I'll submit a PR to allow for that to the library?)